src/asciiRegex.js

```diff
@@ -10,8 +10,6 @@ const names = flatten(
   })
 ).join("|");
 
-const edgeCases = ["http", "https"].join("|");
```

Comment:
I also think I would like to keep this. Instead of just `startOrURL`, we can have `const edgeCases = ["https?\\S*"].join("|");`. That way it's ready for future other things we would like to blacklist and it matches with the business logic at https://github.com/tommoor/react-emoji-render/blob/master/src/renderer.js#L67.